### ISSUE TITLE
Decouple the UI code

### ISSUE THREAD

**Issue by CryoByte33** [CryoByte33/steam-deck-utilities#81]
The `internal` package is a mess, split it into several subpackages:
* UI
* CLI
* Util (or something of the sort)

This is a prerequisite for reconsidering the GUI library being used.

**Comment by nbstrong** [comment]
Frankly, I think the UI should be a separate repo that calls the CLI software as a submodule.

It's a huge pain trying to install this via ssh into the deck when it tries to open windows.

**Comment by CryoByte33** [comment]
I disagree, the core intention is for non-power users to have access to these tweaks, of which is almost entirely the GUI presentation.

Your specific issue is a problem with the installation script, which doesn't need to have a UI and should have a CLI-only option. In the meantime, I provided a manual install process that circumvents that issue entirely.

**Comment by nbstrong** [comment]
This one? https://github.com/CryoByte33/steam-deck-utilities/blob/main/docs/manual-install.md

That one: 
* requires you to open a browser and download files
* makes a bunch of desktop icons
* asks you to make a directory .cryo_utilities and then put everything in /home/deck/cryo_utilities
* calls xdg-icon-resource install cryo-utilities.png --size 64 but there is no cryo-utilities.png

**Comment by CryoByte33** [comment]
Correct, I believe that it includes downloading the icon but if not then I'll need to fix it.

As for cloning the repo and running the install script, I disagree for the "normal" script, and the manual process was actually just made in case GitHub was having issues, which it has frequently.

Again, this tool is made primarily for novice users who don't understand (or don't want to) run commands of any sort, but I'll do my best to make it a bit simpler for others. The SSH/CLI use case will never be primary, though.
